ncm-download/src/main/perl/download.pm

```diff
@@ -80,7 +80,9 @@ sub get_gss_token
 
     # Assume "kinit" is in the PATH.
     my $errs = "";
-    my $proc = CAF::Process->new(["kinit", "-k"],
+    my @kinit_cmd = ("kinit", "-k");
+    push(@kinit_cmd, split(/ /, $tree->{kinit_args})) if ($tree->{kinit_args});
```

Comment:
the `kinit_args` should be a list of strings, and have to be added to the schema.
@natari do you expect different args per file or not? if you do, we cannot use the cache